contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -39,6 +40,11 @@ def __init__(
       this value to True.
     :param style_ignore_path: relative path to file specifying patterns of files to ignore. The syntax
       supported is the same as the .eslintignore/.gitignore syntax.
+    :param bin_executables: A map of executable names to local file name. If a single executable is
```

Comment:
This could probably use an example in the help text here (note that this help text renders on the website, so it's useful to invest in it).

Reply:
done